### DISCUSSION TITLE
How can I launch apps from overview or dock with options?

### DISCUSSION THREAD

**Discussion by lVentus** [end-4/dots-hyprland#589] [Q&A]
Hello!
Like when I use some apps, I want to use option like this:
bilibili --ozone-platform=wayland --enable-wayland-ime
it can launch the app in wayland mode and I can use my fcitx5 input.
But when I wanted to pin a app with options, it doesn't show.
when I launch from overview (as a launcher), it also launch it without any options.
Anyway I can set it up?
Thanks

**Comment by clsty** [comment]
Copy desktop file from `/usr/share/applications/` to `~/.local/share/applications/`, then edit value of `Exec` for the desktop file.

See https://wiki.archlinux.org/title/Desktop_entries
